mesonbuild/compilers/fortran.py

```diff
@@ -392,7 +392,7 @@ def openmp_flags(self, env: Environment) -> T.List[str]:
         return ['-xopenmp']
 
 
-class IntelFortranCompiler(IntelGnuLikeCompiler, FortranCompiler):
```

Comment:
Mypy is pointing at this, and I think it's correct, dropping this is changing the compiler. I think this is also used for the old `ifort` compiler, so dropping this code will absolutely break that compiler